### ISSUE TITLE
when resolution comes to 1280*720 recorded fps comes to 10

### ISSUE THREAD

**Issue by jw20082009** [wysaid/android-gpuimage-plus#329]
If only there MediaCodec is used, there should be a very big promotion. I have ran this library on a Huawei p20，then the recorded video still be a little bit nonfluency. And fps might be 17. But when I ran this on any other machine , I have no eyes to see the recorded video.   I strongly recommended if there anyone who can , the MediaCodec should be.

**Comment by wysaid** [comment]
Yes, I will try to do that.

**Comment by jw20082009** [comment]
maybe I can help you.

**Comment by tsoumalis** [comment]
Is there any progress in the MediaCodec implementation? I recently read that FFmpeg supports it if built with --enable-jni and --enable-mediacodec so maybe you could take advantage of that and still use FFmpeg with little modifications.

**Comment by jw20082009** [comment]
@tsoumalis I had seen many people written about this with decoding but not encoding，thought I am not sure about if this is working with encoding.

**Comment by tsoumalis** [comment]
@jw20082009 Yes I think you are right, everything I can find is about decoding. Maybe encoding will also be supported in the future but I cannot find if there is a timeline for that.

**Comment by huunhan18pro** [comment]
Hi @jw20082009 , I got the same issue. Did you resolve it?
